schema/model_pointset_point.json

```diff
@@ -33,6 +33,14 @@
         "failure"
       ]
     },
+    "range_min": {
+      "description": "Represents the lower bound of the error threshold for a point",
+      "examples": [0]
```

Comment:
type?  should be number?

Reply:
yup, fixed

Reply:
Did you commit/push all changes?  The schema/ file isn't showing that it's been changed, but the gencode/ file is updated... ??!?

Reply:
I missed the schema file. Committed now.

Reply:
The irony is that with a small PR I'd notice something like that, but with a big PR I wouldn't!

Approved.